chainer/links/caffe/caffe_function.py

```diff
@@ -384,6 +384,28 @@ def _setup_scale(self, layer):
         self.forwards[layer.name] = _CallChildLink(self, layer.name)
         self._add_layer(layer)
 
+    @_layer('Slice', 'SLICE')
+    def _setup_slice(self, layer):
+        if layer.slice_param.HasField('axis'):
+            axis = layer.slice_param.axis
+        elif layer.slice_param.HasField('slice_dim'):
+            axis = layer.slice_param.slice_dim
+        else:
+            axis = 1
+
+        if layer.slice_param.slice_point == []:
+            indices_or_sections = len(list(layer.top))
+        else:
+            indices_or_sections = list(layer.slice_params.slice_point)
```

Comment:
Could you replace ``layer.slice_params.slice_point`` with ``layer.slice_param.slice_point``?